CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Allow enumerating objects within the migration they get deleted in.
```

Comment:
This is a bug fix and not an enhancement. It's also not an accurate description of the change; it allows enumerating objects of types which are no longer present in the schema, which is unrelated to objects being deleted.